qemu/tests/cfg/pcie_hotplug_opt.cfg

```diff
@@ -1,5 +1,5 @@
 - pcie_hotplug_opt:
-    only q35
+    only q35 aarch64
```

Comment:
`q35` and `aarch64` are different levels, so suggest split into 2 lines.

Reply:
Ok, corrected.